lang/src/OTS/IMP/org/partiql/ots/legacy/plugin/StandardPlugin.kt

```diff
@@ -22,16 +42,14 @@ import OTS.ITF.org.partiql.ots.operator.LikeOp
 import OTS.ITF.org.partiql.ots.operator.NegOp
 import OTS.ITF.org.partiql.ots.operator.NotOp
 import OTS.ITF.org.partiql.ots.operator.PosOp
+import OTS.ITF.org.partiql.ots.type.BoolType
 import com.amazon.ion.Timestamp
 import java.time.ZoneOffset
 
 data class StandardPlugin(
```

Comment:
Separately, let's switch from the "standard" to the "legacy" lexeme here as well.